mypy.ini

```diff
@@ -1,13 +1,2 @@
 [mypy]
 ignore_missing_imports = True
-exclude = (?x) (
-  diffing.py$ |
-  polytracker/grammars.py$ |
-  test_bb_splitting.py$ |
-  tests/test_grammar_extraction_python.py$ |
-  tests/test_grammar_extraction_native.py$
-  )
-
-[mypy-polytracker.mimid.*]
-# Mimid upstream has lots of type errors, so ignore them
```

Comment:
The only place I see Mimid mentioned is here - https://github.com/trailofbits/polytracker/search?q=mimid . Was it a prior dep?

Reply:
Yes, I believe it was related to the grammar recovery previously attempted. It is not used anymore.